### ISSUE TITLE
idea-runner undefined setting with v3.0.0

### ISSUE THREAD

**Issue by regadas** [JetBrains/sbt-idea-plugin#29]
```scala
lazy val ideaRunner: Project =
  createRunnerProject(plugin, "idea-runner")
```

```
[error] Reference to undefined setting: 
[error] 
[error]   idea-runner / ideaMainJars from idea-runner / Compile / unmanagedJars (Utils.scala:22)
[error]  
[error] Use 'last' for the full log.
```

**Comment by mutcianm** [comment]
Could you please post the whole build definition?
Couldn't reproduce with HOCON plugin as an example

**Comment by mutcianm** [comment]
Okay, I reproduced the issue eventually. `SbtIdeaPlugin` isn't enabled automatically for generated runner project

**Comment by mutcianm** [comment]
fixed via 62f6b0f60119a55c0293856e595de0a39a37323e
